get-request.md

```diff
@@ -5,6 +5,14 @@ The GET gives the customer to access the burger meal section of the online menu
 | --- | --- |
 | GET | base_url://lunch/bugerMeal |
 
+
```

Comment:
In line 2, either change to

This request gives ... or
The GET/lunch request gives ...